src/main/java/org/prebid/server/proto/openrtb/ext/request/yahooads/ExtImpYahooAds.java

```diff
@@ -0,0 +1,16 @@
+package org.prebid.server.proto.openrtb.ext.request.yahooads;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+/**
+ * Defines the contract for bidRequest.imp[i].ext.yahooAds
+ */
```

Comment:
We are trying to remove these comments in new versions, 
so please get rid of it